packages/core/src/discovery.js

```diff
@@ -76,11 +76,11 @@ function debugSnapshotOptions(snapshot) {
 }
 
 // Wait for a page's asset discovery network to idle
-function waitForDiscoveryNetworkIdle(page, options) {
+function waitForDiscoveryNetworkIdle(page, options, captureResponsiveAssetsEnabled) {
```

Comment:
isnt that already part of `options` ?  if not as discovery is passed you can just add it under discovery options. - it should already be there regardless